utils/sim/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_executable(mos-sim fake6502.c mos-sim.c)
```

Comment:
Do you have a paired PR in mind for llvm-test-suite? This will probably break the continuous test suite runs, so we should try to merge both of those at the same time. No action required though, since this is still a draft.

Reply:
The only necessary change to the test suite is the `README`.

Here are the cmake variables I've been using to test (prefix added to PATH):

```
-DCMAKE_C_COMPILER=mos-sim-clang -DCMAKE_CXX_COMPILER=mos-sim-clang++ -DARCH=MOS -DTEST_SUITE_SUBDIRS=SingleSource/ -DTEST_SUITE_RUN_UNDER=mos-sim -DTEST_SUITE_USER_MODE_EMULATION=yes -DTEST_SUITE_LLVM_SIZE="" -DCMAKE_STRIP="" -C../cmake/caches/Os.cmake -G Ninja ..
```

Reply:
What about the github actions for the continuous test runs? These include the literal command lines used to cmake the test suite:
https://github.com/llvm-mos/llvm-test-suite/tree/main/.github/workflows

Reply:
Oh right, I'll prepare a PR for that.